plasma_core/utils/eip712_struct_hash.py

```diff
@@ -0,0 +1,54 @@
+from eip712_structs import EIP712Struct, Address, Uint, Bytes, make_domain
+from eth_utils.crypto import keccak
+from plasma_core.constants import NULL_HASH
+
+
+def hash_struct(tx, domain=None):
+    inputs = [Input(blknum=i.blknum, txindex=i.txindex, oindex=i.oindex) for i in tx.inputs]
+    outputs = [Output(owner=o.owner, token=o.token, amount=o.amount) for o in tx.outputs]
+
+    domain = domain or make_domain(
+        name='OMG Network',
+        version='1',
+        chainId=4,
```

Comment:
please remember to refactor this chain id as well (set the same value as it will be sent at contract creation)

Reply:
I'm leaving it for #98